contracts/child/ChildERC721.sol

```diff
@@ -15,14 +16,19 @@ contract ChildERC721 is ChildToken, ERC721Full {
   );
 
   // constructor
-  constructor (address _token, string name, string symbol) ERC721Full(name, symbol)
+  constructor (address _owner, address _token, string name, string symbol) ERC721Full(name, symbol)
     public 
     {
-    require(_token != address(0));
-
+    require(_token != address(0x0) && _owner != address(0x0));
+    parentOwner = _owner;
```

Comment:
`parentOwner` or `parent`?

Reply:
`parentOwner` is an owner of parent Contract.
`parent` is the address of parent contract.
show we rename ?